Source/Fuse.Elements/Element.Drawing.uno

```diff
@@ -66,21 +66,17 @@ namespace Fuse.Elements
 			if (Visibility != Visibility.Visible)
 				return;
 
+			var visibleRect = GetVisibleViewportInvertPixelRect(dc, RenderBoundsWithEffects);
+			if (visibleRect.Size.X == 0 || visibleRect.Size.Y == 0)
+				return;
+
 			extern double t;
 			if defined(FUSELIBS_PROFILING)
 			{
 				t = Uno.Diagnostics.Clock.GetSeconds();
 				Fuse.Profiling.BeginRegion(this.ToString());
 			}
 
-			var visibleRect = GetVisibleViewportInvertPixelRect(dc, RenderBoundsWithEffects);
-			if (visibleRect.Size.X == 0 || visibleRect.Size.Y == 0)
-			{
-				if defined(FUSELIBS_PROFILING)
-					Fuse.Profiling.EndRegion(Uno.Diagnostics.Clock.GetSeconds() - t);
-				return;
-			}
-
```

Comment:
The old implementation makes the profiler borderline useless in the very common use case of having a long list of nodes where most of the nodes are outside the viewport. The profiler has a not insignificant overhead per node of data it needs to report back home.